resources/config/snemo/demonstrator/simulation/primary_events/1.3/README.rst

```diff
@@ -0,0 +1,178 @@
+      --list-print-mode "raw"
+..
+
+Using variants (there is no difference here):
+
+.. raw:: sh
+
+   $ bxgenbb_inspector \
+      --datatools::resource-path "falaise@$(pwd)/BuildProducts/share/Falaise-3.1.1/resources" \
+      --variant-config "@falaise:config/snemo/demonstrator/simulation/primary_events/1.3/variants/repository.conf" \
+      --variant-gui \
+      --configuration "@falaise:config/snemo/demonstrator/simulation/primary_events/1.3/manager.conf" \
+      --action "list" \
+      --list-print-mode "raw"
+..
+
+Generate 1000 primary events from the ``Bi214_Po214`` generator without variant
+==================================================================================
+
+From Falaise build directory, run:
+
+.. raw:: sh
+
+   $ bxgenbb_inspector \
+      --datatools::resource-path "falaise@$(pwd)/BuildProducts/share/Falaise-3.1.1/resources" \
+      --configuration "@falaise:config/snemo/demonstrator/simulation/primary_events/1.3/manager.conf" \
+      --action "shoot"  \
+      --generator "Bi214_Po214" \
+      --prng-seed 314159 \
+      --number-of-events 1000 \
+      --modulo 100 \
+      --histo-def "@genbb_help:inspector/config/le_nuphy-1.0/inspector_histos_prompt.conf" \
+      --histo-def "@genbb_help:inspector/config/le_nuphy-1.0/inspector_histos_delayed.conf" \
+      --prompt \
+      --delayed \
+      --prompt-time-limit 1 \
+      --title-prefix "Bi214_Po214" \
+      --output-file "histos_Bi214_Po214.root"
+..
+
+Then use Root to browse the generated histograms:
+
+.. raw:: sh
+
+   $ root histos_Bi214_Po214.root
+   ...
+   root [0]
+   Attaching file histos_Bi214_Po214.root as _file0...
+   root [1]  TBrowser b; // Here you may browse the histograms
+   root [2] .q
+..
+
+Generate 1000 primary events from the ``gamma.1MeV`` generator from the variant setup
+=======================================================================================
+
+From Falaise build directory, run:
+
+.. raw:: sh
+
+   $ bxgenbb_inspector \
+      --datatools::resource-path "falaise@$(pwd)/BuildProducts/share/Falaise-3.1.1/resources" \
+      --variant-config "@falaise:config/snemo/demonstrator/simulation/primary_events/1.3/variants/repository.conf" \
+      --variant-set "primary_events:generator=gamma.1MeV" \
+      --variant-gui \
+      --configuration "@falaise:config/snemo/demonstrator/simulation/primary_events/1.3/manager.conf" \
+      --action "shoot"  \
+      --prng-seed 314159 \
+      --number-of-events 1000 \
+      --modulo 100 \
+      --histo-def "@genbb_help:inspector/config/le_nuphy-1.0/inspector_histos_prompt.conf" \
+      --prompt \
+      --title-prefix "gamma_1MeV" \
+      --output-file "histos_gamma_1MeV.root"
+..
+
+Then use Root to browse the generated histograms:
+
+.. raw:: sh
+
+   $ root histos_gamma_1MeV.root
+   ...
+   root [0]
+   Attaching file histos_Bi214_Po214.root as _file0...
+   root [1]  TBrowser b; // Here you may browse the histograms
+   root [2] .q
+..
+
+.. end
```

Comment:
This file should document how to develop and test the resource files, not use them.

Modulo the versions, it may also be better to unify this file with the `primary_events/README` file to avoid confusion.

Reply:
It is not the spirit of this file.
This file is  a memo for developpers/experts only.  This is not documentation for standard users.
It may give hints to users but stuff in it are not supposed to be used as is.
Such notes helped me to manually test and validate the new config and then
build the test scripts integrated in the build system.

primary_events/README is more for users and its content can be used to 
create some documentation.

Reply:
Then make this explicit in the file! Even for developers, the instructions don't tell you (or link to relevant documentation in Bayeux) what the results of running the various programs should be, or how to develop and test new setups.